File: BlazorApp1/Estabelecimentos.Modelos/Estabelecimento.cs
﻿namespace ControleDeEstabelecimentos.Modelos
{
    public class Estabelecimento
    {
        protected string nome;
        private int cnpj;
        protected string descricao;
        protected string logo;
     
        public int Id { get; set; }
        public string Nome { get { return this.nome; } set { this.nome = value; }}
        public int Cnpj { get { return this.cnpj; } set { this.cnpj = value; } }
        public string Descricao { get { return this.descricao; } set { this.descricao = value; } }
        public string Logo { get { return this.logo; } set { this.logo = value; } }

    }
}

File: FilaOnline/Model/Estabelecimento.php
<?php
class Estabelecimento
{
    // Atributos
    public $id;
    private $nome;
    private $email;
    private $cnpj;
    private $endereco;
    private $descricao;
    private $logo;
    private $senha;

     // Getters
     public function getId() 
     {
         return $this->id;
     }
     public function getNome() 
     {
         return $this->nome;
     }
     public function getEmail() 
     {
         return $this->email;
     }
     public function getCnpj() 
     {
         return $this->cnpj;
     }
     public function getEndereco() 
     {
         return $this->endereco;
     }
     public function getDescricao() 
     {
         return $this->descricao;
     }
     public function getLogo() 
     {
         return $this->logo;
     }
     public function getSenha() 
     {
         return $this->senha;
     }
      
     // Setters
     public function setId($id) 
     {
         $this->id = $id;
     }
     public function setNome($nome) 
     {
         $this->nome = $nome;
     }
     public function setEmail($email) 
     {
         $this->email = $email;
     }
     public function setCnpj($cnpj) 
     {
         $this->cnpj = $cnpj;
     }
     public function setEndereco($endereco) 
     {
         $this->endereco = $endereco;
     }
     public function setDescricao($descricao) 
     {
         $this->descricao = $descricao;
     }
     public function setLogo($logo) 
     {
         $this->logo = $logo;
     }
     public function setSenha($senha) 
     {
         $this->senha = $senha;
     }
}
?>